payment/models.py

```diff
@@ -109,50 +109,57 @@ def save(self):
                        ex=self.PAY_STORE_TIME)
 
 
-class Watcher(ModelWithStr):
-    # TODO: Replace wallet 'life' with a specific list of order id's to watch
-    wallet_addresses = DictType(IntType)  # Address : 'life' (how many remove requests before deletion)
-    callback = StringType()  # a webhook to call when a payment is complete
-    service_id = StringType()
+class Watcher():
+    @classmethod
+    def add(cls, service_id, address, order_id):
+        # Add an order to the address, redis will not add the same order_id twice.
+        redis_conn.sadd(service_id + ':' + address, order_id)
```

Comment:
"Landmine" :)  
Can we create a method that generates this string and use it everywhere?